src/Czechia/Mews.Fiscalizations.Czechia/Communication/SoapSigner.cs

```diff
@@ -32,6 +34,7 @@ public SoapSigner(Certificate certificate, SignAlgorithm signAlgorithm)
 
         public XmlDocument SignMessage(XmlDocument xmlDoc)
         {
+            CryptoConfig.AddAlgorithm(typeof(RSAPKCS1SHA256SignatureDescription), "http://www.w3.org/2001/04/xmldsig-more#rsa-sha256");
```

Comment:
It would be nicer if we could manage it without this global setting. But after some quick searching, I also didn't find a solution.